api/serverpb/server.proto

```diff
@@ -351,17 +308,6 @@ service Server {
       description: "Changes PMM Server settings."
     };
   }
-  // TestEmailAlertingSettings sends test email to check current SMTP settings for email alerting.
-  rpc TestEmailAlertingSettings(TestEmailAlertingSettingsRequest) returns (TestEmailAlertingSettingsResponse) {
-    option (google.api.http) = {
-      post: "/v1/Settings/TestEmailAlertingSettings"
-      body: "*"
-    };
-    option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
-      summary: "Test email alerting"
-      description: "Sends test email to check current SMTP settings for email alerting."
-    };
-  }
```

Comment:
It's breaking change, but it has no sense to keep this endpoint

Reply:
Let's link a doc ticket to PMM-11079 so we mention it in RN.